ProgGuide/commands.rst

```diff
@@ -548,7 +548,10 @@ If YottaDB cannot create the process because of something that is unlikely to ch
 The JOB Environment
 +++++++++++++++++++++++++++++
 
-When the JOB is forked, UNIX creates the environment for the new process by copying the environment of the process issuing the JOB command and making a few minor modifications. By default, the standard input is assigned to the null device, the standard output is assigned to routinename.mjo, and the standard error is assigned to routinename.mje.
+When the JOB is forked, UNIX creates the environment for the new process by copying the environment of the process issuing the JOB command and making a few minor modifications. By default, the standard input is assigned to the null device, the standard output is assigned to routinename.mjo, and the standard error is assigned to routinename.mje. 
```

Comment:
When the JOB is forked, UNIX creates the new process -> YottaDB creates the environment for the new jobbed off process